### DISCUSSION TITLE
Importing Bookmarks/Passwords from Mozilla Firefox

### DISCUSSION THREAD

**Discussion by ghost** [qutebrowser/qutebrowser#6829] [Q&A] [answered]
Hello! I just installed QuteBrowser, lured by promises of vim-like keybindings and seamless browsing (which haven't disappointed). However, my passwords are located in Mozilla Firefox, so I'm currently unable to login without copy/paste directly from Firefox... Could these passwords be imported, per chance? Much appreciated.

**Comment by The-Compiler** [comment, marked as answer]
For bookmarks, see [importer.py](https://github.com/qutebrowser/qutebrowser/blob/master/scripts/importer.py). As for passwords, qutebrowser doesn't do password management. It has [various userscripts](https://github.com/qutebrowser/qutebrowser/tree/master/misc/userscripts) to integrate with password managers, you might find a tool to export to one of those.
